### PR TITLE
feat(add_project_admin): Added administration page of create project

### PR DESCRIPTION
**Pull request by amritkv**
Feature : Added administration page of add project page

### CODE REVIEW

File: src/components/ProjectAddSummary/component/Administration/LicenseInfoHeader.tsx [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+    return (
+        <>
+            <OverlayTrigger overlay={<Tooltip>{text}</Tooltip>}>
+                <span className='d-inline-block'>
+                    <BiInfoCircle />
+                </span>
+            </OverlayTrigger>
+        </>
+    )
+}
```
Comment (heliocastro) [inline src/components/ProjectAddSummary/component/Administration/LicenseInfoHeader.tsx:27]: Can this specific one become a global component ?

Comment (heliocastro) [reply src/components/ProjectAddSummary/component/Administration/LicenseInfoHeader.tsx:27]: You are duplicating it in more than two sources

**Review by heliocastro — CHANGES_REQUESTED** [review]
@amritkv I think you have a global component on your hands

File: src/components/ProjectAddSummary/component/Administration/LicenseInfoHeader.tsx (hunk 1 above)
Comment (amritkv) [reply src/components/ProjectAddSummary/component/Administration/LicenseInfoHeader.tsx]: Made a global component for ShowInfoOnHover & resolved in all occurrences.

Comment (amritkv) [reply src/components/ProjectAddSummary/component/Administration/LicenseInfoHeader.tsx]: Made a global component for ShowInfoOnHover & resolved in all occurrences.

**Review by heliocastro — APPROVED** [review]
LGTM

**Pull request merged**